### PR TITLE
Add setup.py to handle pypizing README

### PR DESCRIPTION
**Pull request by jerry-skydio**
Scrub the README and replace local links with global links
that pypi can access. Remove the extra dark mode tags that pypi
can't recognize.

Topic: setuppy
Reviewers: aaron, brian-k

### CODE REVIEW

**Comment by jerry-skydio** [comment]
Reviews in this chain:
└**https://github.com/Skydio/revup/pull/48** **Replace setup.cfg with setup.py**

**Comment by jerry-skydio** [comment]
| # | head | base | diff | date | summary |
| - | - | - | - | - | - |
| 0 | [f4dc9ffc](/Skydio/revup/commit/f4dc9ffc1a365ac29b5504432b8ea9e755030eb9) | [b8abc845](/Skydio/revup/commit/b8abc845296bbf0f59051f779510552c2e16f223) | [diff](/Skydio/revup/compare/b8abc845296bbf0f59051f779510552c2e16f223..f4dc9ffc1a365ac29b5504432b8ea9e755030eb9) | Sep 16 7:58 PM |  4 files changed, 140 insertions(+), 65 deletions(-) |

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import re
+import subprocess
+from pathlib import Path
+
+from setuptools import find_packages, setup
+
+with open("revup/__init__.py") as f:
+    exec(f.read())
```
Comment (brian-kubisiak-skydio) [inline setup.py:8]: seems unnecessary?

**Review by brian-kubisiak-skydio — COMMENTED** [review]
Noting that `setup.cfg` is the preferred way to configure projects where possible. From the setuptools [docs](https://setuptools.pypa.io/en/latest/userguide/quickstart.html#transitioning-from-setup-py-to-setup-cfg):
> New projects are advised to avoid setup.py configurations (beyond the minimal stub) when custom scripting during the build is not necessary. Examples are kept in this document to help people interested in maintaining or contributing to existing packages that use setup.py. Note that you can still keep most of configuration declarative in setup.cfg or pyproject.toml and use setup.py only for the parts not supported in those files (e.g. C extensions). See note.

And:

> We also recommend users to expose as much as possible configuration in a more declarative way via the pyproject.toml or setup.cfg, and keep the setup.py minimal with only the dynamic parts (or even omit it completely if applicable).

I think it's possible/best practice to declare all the static parts with `setup.cfg` and only use `setup.py` for the dynamic part (`long_description=fixed_readme()`. My understanding of [distutils](https://docs.python.org/3/distutils/configfile.html) makes me think that this should work since `setup.cfg` would override `setup.py` when applicable.

**Review by aaron-skydio — APPROVED** [review]
Agree on putting the stuff that can be purely declarative in `setup.cfg` / `pyproject.toml`, otherwise LGTM

**Pull request merged**